tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array_fill_test.bal

```diff
@@ -269,3 +269,28 @@ function testSingletonTypeArrayFill(int index) returns One[] {
     ar[index] = 1;
     return ar;
 }
+
+function testIllegalArrayInsertion(int index) returns (int|string|boolean)[] {
```

Comment:
Shall we update these tests to define the index within the function instead of passing it as an argument?